pkg/cli/root.go

```diff
@@ -16,6 +16,7 @@ type AppSettings struct {
 	EnvPrefix   string // environment variable MYAPP_.....
 	GitVersion  string
 	GitRevision string
+	LogLevel    string // Global log level for the app
```

Comment:
If we add this. Do we have a backwards compatible path?
Are missing params in the yaml acceptable or it will throw an error?

Reply:
The new parameter is optional. By default the logger is initialized using info level and json encode. Just if LogLevel is defined then the logger is updated, https://github.com/Zondax/golem/pull/150/files#diff-fa1c0f512f57498ed99a3473226bacfba1be2c794c4a1ff08758fd53c9a338b8R72

Reply:
Nice